src/main/java/seedu/address/model/affiliation/AffiliationModifier.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model.affiliation;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Set;
+
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * AffiliationModifier modifies affiliation.
+ */
+public class AffiliationModifier {
+
+    /**
+     * Adds the affiliated person to the affiliations for every
+     * person's affiliation in the affiliation set provided.
+     *
+     * @param affiliationSet The affiliation set that contains all person that needs to add the affiliated person.
+     * @param affiliatedPerson The affiliated person to add to every person in the affiliation set.
+     * @param model The current model that is running.
+     */
+    public static void addAffiliations(Set<Affiliation> affiliationSet, Person affiliatedPerson, Model model) {
```

Comment:
Will it be cleaner if we remove the `affiliationSet` from the parameters and derive it from the `affiliatedPerson` within the function?

Reply:
It is possible, I was thinking to preserve the current method signature to standardize with below method signature due to the reason below.